examples/with-react-native-web/package.json

```diff
@@ -6,7 +6,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "latest",
+    "next": "5.0.1-canary.2",
```

Comment:
Next 5.1 will be out soon, lets keep it as is for now.

Reply:
Ok sure :)